### DISCUSSION TITLE
Show custom fields in Directus tables and allow filtering

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#15674] [Feature Requests]
Having many roles, I added a custom field to the Directus_roles collection.
It works fine.
But

- being able to see the custom field when creating the role (we only see Role name / and App / Admin Access)
- to add the custom field to see it in the roles table (*)
- and use it in filters on the roles table 

would be nice.

(*) allowing a preset. The drop-down on the preset collection shows Directus Activity / Files / Users, but not Roles

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
